oidc_example/op2/requirements.txt

```diff
@@ -0,0 +1,38 @@
+alabaster==0.7.12
+asn1crypto==0.24.0
+backports.functools-lru-cache==1.5
+Beaker==1.10.0
+certifi==2018.11.29
+cffi==1.11.5
+chardet==3.0.4
+cheroot==6.5.4
+CherryPy==8.9.1
+contextlib2==0.5.5
+cryptography==2.5
+defusedxml==0.5.0
+enum34==1.1.6
+funcsigs==1.0.2
+future==0.17.1
+idna==2.8
+ipaddress==1.0.22
+isodate==0.6.0
+jaraco.functools==2.0
+lxml==4.3.0
+Mako==1.0.7
+MarkupSafe==1.1.0
+more-itertools==5.0.0
+oic==0.15.0
```

Comment:
If you merge latest changes from `fix-op2` branch into yours, you should be able to trim this requirements list down to say... `CherryPy==8.9.1` and `oic`. Try it out; 

```bash
python3 -m venv venv
. venv/bin/activate
pip install oic CherryPy==8.9.1
pip freeze > requirements.txt
```

Reply:
(Also add first 3 steps as a nice getting started section such as [this one](https://github.com/mkdevops-se/hq.mkdevops.se#getting-started).)